Parsers/ASimWebSession/Parsers/vimWebSessionBarracudaWAF.yaml

```diff
@@ -1,7 +1,7 @@
 Parser:
```

Comment:
Please check below and map:
(1) Warning: Empty value in 320 records (100.0%)  in mandatory field [EventEndTime] (Schema:WebSession)
(1) Warning: Empty value in 320 records (100.0%)  in mandatory field [EventStartTime] (Schema:WebSession)

Reply:
As data is not there in workspace closing this comments